modelgauge/prompt_formatting.py

```diff
@@ -13,5 +13,4 @@ def format_chat(
         else:
             role_text = sut_role
         blocks.append(f"{role_text}: {message.text}")
-    blocks.append(f"{sut_role}: ")
```

Comment:
Was this a bug? This basically appends the name of the sut role and nothing else at the end of every formatted prompt.

previously:
```
User: hello
Agent: hi there
Agent: 
```

after fix
```
User: hello
Agent: hi there
```

Reply:
Hmm actually, based on the tests file, it seems like this is desired behavior. Do you know why? I'll create a separate method to avoid conflicting

Reply:
This function formats a chat prompt as a text string for completion SUTs. The final _Agent:_ is meant to indicate to the SUT that it's their "turn". Without it, some SUTs may attempt to follow the prompt format and begin their completion with _Agent:_.